### PR TITLE
Use udiff format in CI

### PR DESCRIPTION
**Pull request by julienfalque**

### CODE REVIEW

**Comment by keradus** [comment]
what about setting `PHP_CS_FIXER_FUTURE_MODE=1` instead ?

**Comment by SpacePossum** [comment]
we want non-future to be tested as well, so maybe for one job on maybe like PHP nightly that makes sense?

**Comment by julienfalque** [comment]
Was about to say the same, wouldn't using `PHP_CS_FIXER_FUTURE_MODE` change other things we don't want to change in CI builds?

**Comment by keradus** [comment]
huh? following that reasoning I could say "we want non-udiff differ be tested as well"
but this PR is changing options passed to executing fix command, not tests. non-udiff differ is still tested. same, setting FUTURE env for fix command won't affect tests.

**Comment by julienfalque** [comment]
It's not a reasoning but a real question: I have no idea what setting `PHP_CS_FIXER_FUTURE_MODE` really does right now.

**Comment by julienfalque** [comment]
Ok I checked and indeed it may be a better alternative, PR updated :)

**Comment by keradus** [comment]
as in readme, it set udiff and estimating-max by default

File: .appveyor.yml [hunk 1]
```diff
@@ -39,4 +39,4 @@ before_test:
 test_script:
     - cd C:\projects\php-cs-fixer
     - vendor\bin\phpunit
-    - php php-cs-fixer --diff --dry-run -v fix
+    - php php-cs-fixer --diff-format=udiff --dry-run -v fix
```
Comment (keradus) [inline .appveyor.yml]: this one is still old

Comment (julienfalque) [reply .appveyor.yml]: Using `SOME_VAR=value command` does not work on Windows so I kept using the option instead.

Comment (keradus) [reply .appveyor.yml]: what about 
```
- set PHP_CS_FIXER_FUTURE_MODE=1
- php php-cs-fixer --diff --dry-run -v fix
```
?

File: .appveyor.yml [hunk 2]
```diff
@@ -27,7 +28,6 @@ install:
     - echo extension_dir=ext >> php.ini
     - echo extension=php_curl.dll >> php.ini
     - echo extension=php_openssl.dll >> php.ini
-    - echo extension=php_mbstring.dll >> php.ini
```
Comment (keradus) [inline .appveyor.yml]: is it by purpose ?

Comment (julienfalque) [reply .appveyor.yml]: Nope :)

File: .appveyor.yml (hunk 1 above)
Comment (julienfalque) [reply .appveyor.yml]: Done.

**Comment by keradus** [comment]
Thank you @julienfalque.

**Pull request merged**